lti_consumer/utils.py

```diff
@@ -66,13 +66,20 @@ def get_lms_lti_access_token_link(location):
     )
 
 
-def get_lti_ags_lineitems_url(lti_config_id):
+def get_lti_ags_lineitems_url(lti_config_id, lineitem_id=None):
     """
     Return the LTI AGS endpoint
 
     :param lti_config_id: LTI configuration id
+    :param lineitem_id: LTI Line Item id. Single line item if given an id,
+        otherwise returns list url
     """
-    return "{lms_base}/api/lti_consumer/v1/lti/{lti_config_id}/lti-ags".format(
+    url = "{lms_base}/api/lti_consumer/v1/lti/{lti_config_id}/lti-ags".format(
```

Comment:
Avoid overwriting this variable later.
```suggestion
    base_ags_url = "{lms_base}/api/lti_consumer/v1/lti/{lti_config_id}/lti-ags".format(
```

Reply:
Updated.